lib/iris/tests/__init__.py

```diff
@@ -753,6 +754,21 @@ def tearDown(self):
             plt.close('all')
 
 
+class TestGribMessage(IrisTest):
+
+    def assert_grib_message_contents(self, filename, contents):
```

Comment:
@marqh To align this method name with the same convention used with similar `IrisTest.assertX` methods, would you be swayed into renaming as `assertGribMessage` ?

`IrisTest` method name style examples are, `assertCMLApproxData`, `assertCDL`, `assertCML` etc